tests/testdata/config/byon.yaml

```diff
@@ -27,3 +27,34 @@ spec:
     - route:
       - destination:
           host: byon.test.istio.io
+
+---
+## VM running docker ( see deb/run/docker )
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: byon-docker
+spec:
+   hosts:
+   - byon-docker.test.istio.io
+   ports:
+   - number: 7072
+     name: http
+     protocol: HTTP
+   resolution: STATIC
+   endpoints:
+    - address: 172.18.0.3
+      ports:
+        http: 7072
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: byon-docker
+spec:
+  hosts:
+    - byon-docker.test.istio.io
+  http:
+    - route:
+      - destination:
+          host: byon-docker.test.istio.io
```

Comment:
You don’t need this virtual service. If should work without it.

Reply:
its not IPs. You just dont need this vservice at all. If there is a service entry, there will be a default route

Reply:
Got it, will remove.